lib/templates/webmanifest.js

```diff
@@ -56,6 +60,29 @@ const renderWebManifest = (name, color) => `
 			"sizes": "512x512",
 			"type": "image/png"
 		}
+	],
+	"screenshots": [
```

Comment:
Are we planning on creating portrait screenshots of the different screens on mobile?

Reply:
Not sure what you mean, these are portrait screenshots! It's specified by `"platform": "narrow"` so android know which ones to use. As far as I know no other platforms use screenshots right now. In the future we can add more as needed.

Reply:
I commented before seeing the screenshots 😄 nothing to see here 👻